src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,102 @@
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
+      default:
+        return todos;
+    }
+  };
 
-          <tr data-cy="todo">
-            <td className="is-vcentered">8</td>
+  const list = selectList().filter(l => {
+    return l.title.toLowerCase().includes(query);
+  });
 
+  return (
+    <table className="table is-narrow is-fullwidth">
+      <thead>
+        <tr>
+          <th>#</th>
+
+          <th>
+            <span className="icon">
+              <i className="fas fa-check" />
+            </span>
+          </th>
+
+          <th>Title</th>
+          <th> </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {list.map(todo => (
+          <tr
+            data-cy="todo"
+            key={todo.id}
+          >
+            <td className="is-vcentered">{todo.id}</td>
             <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">quo adipisci enim quam ut ab</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
+              {todo.completed && (
+                <span className="icon" data-cy="iconCompleted">
+                  <i className="fas fa-check" />
                 </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">9</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">molestiae perspiciatis ipsa</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">10</td>
-
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
+              )}
             </td>
 
             <td className="is-vcentered is-expanded">
-              <p className="has-text-success">illo est ratione doloremque quia maiores aut</p>
+              <p
+                className={classNames({
+                  'has-text-danger': !todo.completed,
+                },
+                {
+                  'has-text-success': todo.completed,
+                })}
```

Comment:
One of the possible options: 
```suggestion
                className={todo.completed ? 'has-text-success' : 'has-text-danger'}
```